cms/envs/common.py

```diff
@@ -732,11 +732,6 @@
 DEFAULT_TEMPLATE_ENGINE = TEMPLATES[0]
 
 #################################### AWS #######################################
-# S3BotoStorage insists on a timeout for uploaded assets. We should make it
-# permanent instead, but rather than trying to figure out exactly where that
-# setting is, I'm just bumping the expiration time to something absurd (100
-# years). This is only used if DEFAULT_FILE_STORAGE is overriden to use S3
-# in the global settings.py
```

Comment:
This comment is moved to `cms/envs/production.py`, where the relevant variable was (previously) movied.

Reply:
I don't see this comment moved anywhere.. is it still relevant?

Reply:
A push may have hidden the other comment in this PR, but I updated/simplified it. More detail in the PR comment, but TL;DR: is it's outdated and verbose.

I'm on mobile, but can link it when back at a computer.

Reply:
Ah cool, I'll find it. Thank you!